content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,156 @@
+
+    {{< /note >}}
+
+1. Update all service account tokens to include both old + new CA certificates.
+    
+    If any pods started before new CA is picked by API servers, they will get this update and trust both old + new CAs.
+
+    ```shell
+    base64_encoded_ca="$(base64 <path to file containing both old + new CAs>)"
+
+    for namespace in $(kubectl get ns --no-headers | awk '{print $1}'); do
+        for token in $(kubectl get secrets --namespace "$namespace" --field-selector type=kubernetes.io/service-account-token -o name); do
+            kubectl get $token --namespace "$namespace" -o yaml | \
+            /bin/sed "s/\(ca.crt:\).*/\1 ${base64_encoded_ca}" | \
+            kubectl apply -f -
+        done
+    done
+    ```
+
+1. Restart all pods using in-cluster configs (ex: kube-proxy, coredns, etc) so they can use the updated certificate authority data from *ServiceAccount* secrets.
+
+    1. Make sure coredns and kube-proxy and other pods using in-cluster configs are working as expected.
+
+1. Append new CA to the file against `--client-ca-file` flag in the `kube-apiserver` and `kube-scheduler` configuration.
+
+    {{< note >}}
+
+    If you used `kubeadm` to deploy your cluster, the control plane components run as {{< glossary_tooltip text="static Pods" term_id="static-pod" >}}, managed by the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}}.
+
+    {{< /note >}}
+
+1. Update certificates for user accounts by replacing the content of `client-certificate-data` and `client-key-data` respectively and the `certificate-authority-data` section in all `admin.conf`, `controller-manager.conf`, `scheduler.conf` files with Base64-encoded new CA certificate data:
+
+    For information about creating certificates for individual user accounts, see [Configure certificates for user accounts](/docs/setup/best-practices/certificates/#configure-certificates-for-user-accounts).
+
+1. Restart API servers with the certificates (`apiserver.crt`, `apiserver-kubelet-client.crt`, and `front-proxy-client.crt`) signed by new CA. You can use the existing private keys or new private keys. If you changed the private keys then update these in the Kubernetes certificates directory as well.
+
+    Since the pod trusts both old + new CAs, there will be a momentarily disconnection after which the pod's kube client will reconnect to the new API server that uses the certificate signed by the new CA.
+
+    {{< note >}}
+
+    To generate certificates and private keys for your cluster using the `openssl` command line tool, see [Certificates (`openssl`)](/docs/concepts/cluster-administration/certificates/#openssl).
+    
+    You can also use [`cfssl`](/docs/concepts/cluster-administration/certificates/#cfssl).
+
+    {{< /note >}}
+    
+    1. Restart Scheduler to use the new CAs.
+
+    1. Make sure control plane components logs no TLS errors.
```

Comment:
My suggestion: try changing this to an unordered list and see if that fixes how Hugo builds the site.

Reply:
its fixed, thanks. it was an empty line between 2 sentences inside {{< note >}} shortcode causing issue.